alibi/saving.py

```diff
@@ -165,7 +165,7 @@ def _save_AnchorImage(explainer: 'AnchorImage', path: Union[str, os.PathLike]) -
         dill.dump(segmentation_fn, f, recurse=True)
 
     predictor = explainer.predictor
-    explainer.predictor = None
+    explainer.predictor = None  # type: ignore
```

Comment:
Ignores in this file are purely because we temporarily set some attributes to `None` so that they are not saved to disk. The attributes are reinstated to their original values afterwards.